style.css

```diff
@@ -946,7 +957,7 @@ hr {
 }
 
   .social2{
-    margin-left: 63px;
+    margin-left: 10rem;
```

Comment:
- [ ] [OPTIONAL] I kindly suggest you centralise your social icon 
 
![image](https://github.com/Tshepo45/Portfolio-desktop-version/assets/68823331/752f265c-e804-493a-b259-3807dfd05e70)